lib/engine/game/g_1861/game.rb

```diff
@@ -912,7 +912,7 @@ def operating_order
           minors + majors + [@national]
         end
 
-        def add_neutral_tokens
+        def add_neutral_tokens(_hexes)
```

Comment:
without this, 1861 breaks, as it calls for init_hexes from 1867, which calls add_neutral_tokens with the extra params.

I could use .airity to figure out if 1867 or 61 is calling this method, but it seems messier then this.